### PR TITLE
[ZH]panda -> zef

### PR DESCRIPTION
**Pull request by wenjie1991**
Changes:
1. Updating the module installer to `zef`.
2. Updating the Rakudo Star installation method in Linux.
3. Several fine adjustment in updates of English version.

### CODE REVIEW

File: zh.perl6intro.adoc [hunk 1]
```diff
@@ -2829,11 +2831,11 @@ Perl 6是通用编程语言。 它可以用于处理众多任务，包括：
 
 Perl 6 允许创建和重新分发 *modules*。 每个模块是一组封装的功能，可以在安装后重复使用。
 
-_Panda_ 是 Rakudo Star 中自带的模块管理工具。
+_zef_ 是 Rakudo Star 中自带的模块管理工具。
```
Comment (titsuki) [inline zh.perl6intro.adoc]: The first letter should be capital:
```
_Zef_
```

File: zh.perl6intro.adoc [hunk 2]
```diff
@@ -1460,16 +1461,16 @@ say $squared(9);
 
 我们假设你有一个数组。你被要求返回该数组的唯一值, 并且按从大到小的顺序排序。
 
-你可能会通过写出近似于这样的代码来解决那个问题:
+下面是没有使用链式调用的代码：
```
Comment (titsuki) [inline zh.perl6intro.adoc]: I think this line should use `:` instead of `：`.

File: zh.perl6intro.adoc [hunk 3]
```diff
@@ -85,12 +86,12 @@ Perl 6 是通过该规范测试套件的实现。
...
 * 使用 homebrew 安装: `brew install rakudo-star`
 * 使用 MacPorts 安装: `sudo port install rakudo`
-* 从 http://rakudo.org/downloads/star/ 下载最新的安装器(.dmg 后缀的文件) 
+* 从 https://rakudo.perl6.org/downloads/star/ 下载最新的安装器(.dmg 后缀的文件) 
 
 .Windows
 . 从 http://rakudo.org/downloads/star/ 下载最新的安装器(.msi 后缀的文件) +
 如果你的系统架构是 32-bit, 则下载 x86 文件; 如果是 64-bit, 下载 x86_64 文件。
-. 安装完成后确保 `C:\rakudo\bin` 在 PATH 中。
+. 安装完成后，确保 `C:\rakudo\bin` 在 PATH 中。
```
Comment (titsuki) [inline zh.perl6intro.adoc]: I think this line should use `, ` instead of `，`, because the other lines use `, `.

**Comment by wenjie1991** [comment]
Thank you for correcting the wrong punctuation. Please made the changes you asked for. Thanks!

**Comment by titsuki** [comment]
Thanks for your PR and reply, but sorry, what does "Please made the changes" means ?
Which one is what you want to say ?
+ I see, but I have no time to correct the matters pointed out.
+ I will fix it ASAP. Would you give me a few days to correct the matters.

If you mean the first one, I'll fix the matters myself.

**Comment by wenjie1991** [comment]
Sorry, my fault. I am new to github. I mistakenly assumed you can directly modify my commit.

**Comment by titsuki** [comment]
Thanks a lot !

**Pull request merged**